### PR TITLE
Cockpit 134

### PR DESCRIPTION
**Pull request by dperpeet**
Work in progress for the 134 release notes: https://github.com/cockpit-project/cockpit/releases/tag/134

Should we add notes for the following?

- NetworkManager now activates slave interfaces by itself
- Drop "System" service actions which are intended for scripts

### CODE REVIEW

**Comment by martinpitt** [comment]
> NetworkManager now activates slave interfaces by itself

Judging by the tone and technical level of the release note, I think that this isn't really a change that is very visible to users (who aren't NM developers), it feels more like a bug fix to me. So I'd skip it.

> Drop "System" service actions which are intended for scripts

This is user-visible, so how about something like

### Complex Service actions got dropped

Cockpit previously exposed systemd unit actions such as "Reload or Try Restart" which are only useful for scripts. The action list now got cleaned up to only show actions which are useful for interactive human usage.

**Comment by dperpeet** [comment]
thanks for your feedback, updated

File: content/cockpit-134.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+page now shows the current message of the day. When dismissed by clicking on the ```X``` close button, the message
+is hidden until it changes.
+
+![Message of the day](http://cockpit-project.org/blog/images/cockpit-motd.png)
+
+### Expose fewer system service actions
+
+Cockpit previously exposed systemd unit actions such as ```Reload or Try Restart``` which are primarily intended for
+scripts. The action list now only show actions which are useful for interactive human usage: ```Start```, ```Stop```,
+```Restart``` and ```Reload```. This makes it easier for the user to pick out an action.
```
Comment (martinpitt) [inline content/cockpit-134.md]: Oxford comma missing after Restart.

File: content/cockpit-134.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Cockpit now supports one more feature it was previously missing compared to the command line login: the ```System```
+page now shows the current message of the day. When dismissed by clicking on the ```X``` close button, the message
+is hidden until it changes.
+
+![Message of the day](http://cockpit-project.org/blog/images/cockpit-motd.png)
+
+### Expose fewer system service actions
+
+Cockpit previously exposed systemd unit actions such as ```Reload or Try Restart``` which are primarily intended for
+scripts. The action list now only show actions which are useful for interactive human usage: ```Start```, ```Stop```,
```
Comment (martinpitt) [inline content/cockpit-134.md]: Too many "actions" for my taste, so how about "Now it only shows actions which are useful for interactive human usage:"?

File: content/cockpit-134.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+here is greatly appreciated and a big thank you to our contributors!
+
+[Cockpit translations](https://fedora.zanata.org/project/view/cockpit?dswid=-8859)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/_4jQ6NYSL-c?ecver=1" frameborder="0" allowfullscreen></iframe>
+
+### Show message of the day on the System page
+
+Cockpit now supports one more feature it was previously missing compared to the command line login: the ```System```
+page now shows the current message of the day. When dismissed by clicking on the ```X``` close button, the message
```
Comment (martinpitt) [inline content/cockpit-134.md]: ... day (```/etc/motd```)

File: content/cockpit-134.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 134.
+
+### Login page is now translatable
+
+The majority of Cockpit has long been translatable. Since the login page is a bit of a special case, it required some
+special attention to make that also play well with the language display settings. Take a look at the video below for a demo.
+Check out the Zanata link below, it's very easy to contribute translations to Cockpit.
+At this time our top 3 translations (>90%) are Polish (pl), Ukranian (uk) and Chinese (zh-CN). Every bit of help
```
Comment (martinpitt) [inline content/cockpit-134.md]: Oxforc comma missing after (uk).

Comment (dperpeet) [inline content/cockpit-134.md]: I don't use that comma. =)

File: content/cockpit-134.md (hunk 1 above)
Comment (dperpeet) [inline content/cockpit-134.md]: I don't use that comma.

File: content/cockpit-134.md (hunk 4 above)
Comment (martinpitt) [inline content/cockpit-134.md]: I appreciate the irony in making typos while nitpicking about other people's typos :)

Comment (dperpeet) [inline content/cockpit-134.md]: Update: I only use that comma if it helps avoid ambiguity and/or improves clarity.

File: content/cockpit-134.md (hunk 1 above)
Comment (dperpeet) [inline content/cockpit-134.md]: Update: I only use that comma if it helps avoid ambiguity and/or improves clarity.

File: content/cockpit-134.md (hunk 3 above)
Comment (dperpeet) [inline content/cockpit-134.md]: changed

File: content/cockpit-134.md (hunk 2 above)
Comment (dperpeet) [inline content/cockpit-134.md]: Also removing last sentence, it seems redundant.

**Comment by dperpeet** [comment]
updated

File: content/cockpit-134.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+Cockpit now supports one more feature it was previously missing compared to the command line login: the ```System```
+page now shows the current message of the day (```/etc/motd```). When dismissed by clicking on the ```X``` close button,
+the message is hidden until it changes.
+
+![Message of the day](http://cockpit-project.org/blog/images/cockpit-motd.png)
+
+### Expose fewer system service actions
+
+Cockpit previously exposed systemd unit actions such as ```Reload or Try Restart``` which are primarily intended for
+scripts. The dropdown list now only show actions which are useful for interactive human usage: ```Start```, ```Stop```,
```
Comment (martinpitt) [inline content/cockpit-134.md]: "shows"

**Pull request merged**